.github/workflows/test_agents.yaml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request: {}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - extra: ""
+            script: "test_basic.py"
+            python-version: "3.10"
+          - extra: "stable-baselines"
+            script: "test_sb.py"
+            python-version: "3.7"
+          - extra: "stable-baselines3"
+            script: "test_sb3.py"
+            python-version: "3.10"
+          - extra: "ray-rllib"
+            script: "test_ray_rllib.py"
+            python-version: "3.10"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{matrix.python-version}}
+      - run: python3 -m pip install wheel==0.38.4
+      - run: python3 -m pip install .[tests]
+      - if: ${{ matrix.extra != '' }}
+        run: python3 -m pip install .[${{matrix.extra}}]
+      - if: ${{ matrix.extra == 'stable-baselines' }}
+        run: python3 -m pip install tensorflow==1.14.0
+      - run: git clone https://github.com/diambra/agents.git agents
+      - run: pytest agents/tests/${{matrix.script}}
```

Comment:
here I was not sure how to properly handle directory navigation, so I just cloned the repo in the current dir (so it is cloned inside the arena repo) and used the absolute path to execute the tests, they do not need a specific working dir to work, so they are ok. 

If you think it is a bad thing and are able to suggest a cleaner way, let me know.